VALIDATORS_FAQ.md

```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
 
 The validator that is selected to propose the next block is called proposer. Each proposer is selected deterministically, and the frequency of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 Atoms and a validator's total stake is 10 Atoms, then this validator will be chosen 10% of the time as the next proposer.
 
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+Yes, they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block rewards and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called Photons.
```

Comment:
Yes, they will. Initially, validators of the Cosmos hub will also validate the first public Ethermint zone.  If governance decides so, validators of the Cosmos hub may be required to validate additional zones in the Cosmos ecosystem. As the case with the Ethermint Zone, for each additional zone compensation is to be provided in the form of block rewards and transaction fees.

Reply:
ok